src/components/card/new-horizontal-resource-card.tsx

```diff
@@ -54,8 +54,12 @@ const HorizontalResourceCard: React.FC<{
       tag={resource.tag?.name}
     >
       <Card {...props} resource={resource} className={defaultClassName}>
-        <CardContent className="grid grid-cols-8 gap-5 items-center px-5 py-2">
-          <CardHeader className="col-span-3 flex items-center justify-center">
+        <CardContent className="grid grid-cols-8 gap-5   items-center px-5 py-2">
+          <CardHeader
+            className={`col-span-3 flex items-center justify-center ${cx({
+              '-ml-32 -my-64': resource.name === 'article',
+            })}`}
```

Comment:
if resource is an article we want to position the card to the left and applying negative margin on `y` spreads the image across the card vertically